client/dashboard/sites/features.ts

```diff
@@ -17,6 +22,29 @@ export const HostingFeatures = {
 
 export type HostingFeatures = ( typeof HostingFeatures )[ keyof typeof HostingFeatures ];
 
+export function canManageSite( site: Site ) {
+	if ( site.is_deleted || ! site.capabilities.manage_options ) {
+		return false;
+	}
+
+	// P2 sites are not supported.
+	if ( site.options?.is_wpforteams_site ) {
```

Comment:
Should it be just this?

```suggestion
	if ( isP2( site)  ) {
```